spec/form_builder_spec.rb

```diff
@@ -0,0 +1,422 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+class TestHelper < ActionView::Base; end
+
+RSpec.describe Bali::FormBuilder do
+  let(:helper) { TestHelper.new(ActionView::LookupContext.new(ActionView::PathSet.new), {}, nil) }
+  let(:resource) { Movie.new }
+  let(:builder) { Bali::FormBuilder.new :movie, resource, helper, {} }
+
+  describe '#text_field_group' do
+    it 'renders an input' do
+      expect(builder.text_field_group(:name)).to include(
+        '<div id="field-name" class="field "><label class="label " for="movie_name">Name</label>'\
+        '<div class="control "><input class="input " type="text" name="movie[name]" '\
+        'id="movie_name" /></div></div>'
+      )
+    end
+  end
```

Comment:
I think all this tests should go on its own test file, since they have it own helper file